### PR TITLE
Vagrant update

### PR DESCRIPTION
**Pull request by mkutlak**
See commits

### CODE REVIEW

**Comment by ernestask** [comment]
Would be nice if https://github.com/abrt/abrt-ansible/pull/7/commits/51f25816c00c95b7d708f1f1add4b933e3194ce3 referenced https://github.com/abrt/ansible-role-faf/commit/df236ef8e3210aef67f0990a35dc0e35e3e7c272.

**Comment by ernestask** [comment]
In https://github.com/abrt/abrt-ansible/pull/7/commits/b046ab2a547fc9981c397ed9613717ae0f13d3d9 - “in the past” is really vague. Do you recall anything more?

File: Vagrantfile [hunk 1]
```diff
@@ -48,8 +48,7 @@ Vagrant.configure("2") do |config|
          "all_groups:children" => ["rs"]
        }
        ansible.extra_vars = {
-
-         selinux: false,
+         selinux: true,
```
Comment (ernestask) [inline Vagrantfile]: So retrace-server doesn’t work properly without SELinux in permissive mode, but doesn’t this set it to enforcing?

Comment (ernestask) [reply Vagrantfile]: Oh, my bad, I missed the `selinux_state` variable. Carry on.

**Comment by ernestask** [comment]
Only nitpicks, LGTM.

**Comment by mkutlak** [comment]
> In [b046ab2](https://github.com/abrt/abrt-ansible/commit/b046ab2a547fc9981c397ed9613717ae0f13d3d9) - “in the past” is really vague. Do you recall anything more?

https://github.com/abrt/abrt-ansible/commit/466bec4788273a336c99eb0dee8df27c8e87729d

**Comment by mkutlak** [comment]
I will add references of the past changes to the commits.

**Comment by ernestask** [comment]
Thanks.

**Pull request merged**